app/views/discovery/_carousel_vendor_element.html.haml

```diff
@@ -1,5 +1,5 @@
 %div{:class => "container carousel_element #{local_assigns[:ownership].gsub(" ", "_")}", :id => local_assigns[:vendor].name.gsub(" ", "_"), :onclick => "routeToPageDirectly('#{vendor_path(local_assigns[:vendor].id)}')"}
```

Comment:
@Juggernaut265 shouldn't this be the same as products?

Reply:
Which one is correct? Should I revert my fixes?

Reply:
I thought the "carousel_image.button" was your change, so I got rid of the button. We should keep the one that you originally wrote.